packages/rendermime/src/rendermime.ts

```diff
@@ -363,32 +298,56 @@ namespace RenderMime {
      */
     contents: Contents.IManager;
   }
+
+  /**
+   * Add the default renderer factories to a rendermime instance.
+   */
+  export
+  function addDefaultFactories(rendermime: RenderMime): void {
```

Comment:
This was needed for outside uses of the rendermime (such as our standalone notebook example).